src/main/resources/db/changelog/db.changelog-RDM-2901.xml

```diff
@@ -0,0 +1,38 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="20" author="mohammed.khatri">
+        <sql>
+            CREATE TYPE significant_item_type AS ENUM ('DOCUMENT');
+        </sql>
+        <createTable tableName="case_event_significant_items">
+            <column name="id" type="serial" autoIncrement="true">
+                <constraints nullable="false"/>
+            </column>
+            <column name="description" type="varchar(128)">
```

Comment:
This is defined as a `varchar(128)` but the validation in code is `<=64`. Both should be consistent.

Reply:
Addressed